### ISSUE TITLE
Configuration for "peekable room" in single room mode

### ISSUE THREAD

**Issue by ashfame** [Automattic/chatrix#191]
On Matrix.org it can take a good while (1min+) at times to just get the room state (visibility=world_readable)

So, if we add a configuration along side single room mode (when room is specified), whether the room is peekable or not, we can skip checking for it dynamically and simply fetch messages.

This should speed up the time its required before the timeline can be rendered for the user.

**Comment by psrpinto** [comment]
The problem I see with this is that the state of the room can change (e.g. `history_visibility` can become non-world-readable) after it has been configured.

**Comment by ashfame** [comment]
Yeah! But in order to make that fast, its either caching and then turning out to be incorrect or user supplying that information. Latter seems better. And if we are unable to fetch messages, we can still fail gracefully.
